070-Integrations/030-github.mdx

```diff
@@ -30,7 +30,10 @@ Once permissions are granted you should see a list of GitHub repositories that y
 After connecting a database to a GitHub repository, Xata will create and maintain a preview branch for every pull request that is opened. The preview branch will contain a subset of the data present in the `main` Xata branch, and follow the schema present in the pull request. The process for creating the preview branch is as follows:
 
 - A new preview branch will be created (if it doesn't exist) by forking `main` schema, the preview branch will be named `preview-<owner>-<branch_name>`, where `<owner>` is the user/organization were the pull request base branch is located and `<branch_name>` is the name of the branch.
-- When the preview is created for the first time, part of the data from the `main` branch will be copied over to it. This will be done by copying up to 10,000 rows in the `main` branch.
+- When the preview is created for the first time, part of the data from the `main` branch will be copied over to it. This will be done by copying up to 10,000 rows from the `main` branch.
+<Alert>
+    [File attachments](https://xata.io/docs/sdk/file-attachments) are not copied to the preview branch.
+</Alert>
```

Comment:
I would move this under the title, rather than within the bullet.

Likely this needs to also exist on the workflow docs page as well.

Reply:
I moved the note under the bullet points as well as a subtle comment in the workflow page.

Let me know what you think.